frontend/src/components/form/base/EDatePicker.vue

```diff
@@ -20,6 +20,7 @@ Displays a field as a date picker (can be used with v-model)
   >
     <template #default="picker">
       <v-date-picker
+        :picker-date="pickerMonth"
```

Comment:
Why is the .sync modifier not needed anymore?
(same question for the other places, where the .sync modifier was removed)

Reply:
In vue3 there's a replacement for multiple 2-way-binding properties (argument for v-model):
https://v3-migration.vuejs.org/breaking-changes/v-model

But I guess this is not working in vue2.

Reply:
I remember vaguely that i had some problems with the .sync modifier.
But i could not reproduce it, so i added the .sync modifier, but changed the attribute order.